subprojects/docs/src/docs/release/notes.md

```diff
@@ -305,6 +305,24 @@ task myTask {
 }
 ```
 
+### Calling `TaskInputs` methods with incorrect parameters
+
+`TaskInputs.file()` used to accept anything that `files()` would, and the same was true for `dir()`. Starting with Gradle 4.3 this behavior is deprecated. Do not do this:
+
+```
+task myTask {
+    inputs.dir fileTree(...)
```

Comment:
The deprecation starts with `TaskInputs.file`, but the example mentions the alternative. That might be confusing.